python/txt2tags.py

```diff
@@ -1604,25 +1606,30 @@ def Readfile(file, remove_linebreaks=0, ignore_error=0):
 			if not ignore_error:
 				Error(_('You must feed me with data on STDIN!'))
 	else:
-		try: f = open(file); data = f.readlines() ; f.close()
+		try:
+			f = open(file, encoding='utf-8-sig')
+			data = f.readlines()
+			f.close()
 		except:
 			if not ignore_error:
 				Error(_("Cannot read file:")+" %s"%file)
 	if remove_linebreaks:
-		data = map(lambda x:re.sub('[\n\r]+$','',x), data)
+		data = [re.sub('[\n\r]+$','',x) for x in data]
 	Message(_("File read (%d lines): %s")%(len(data),file),2)
 	return data
 def Savefile(file, contents):
 	try: f = open(file, 'wb')
 	except: Error(_("Cannot open file for writing:")+" %s"%file)
-	if type(contents) == type([]): doit = f.writelines
+	if isinstance(contents, list):
+		doit = f.write
+		contents = (''.join(contents)).encode('utf-8')
```

Comment:
I think I prefer:

```suggestion
		contents = [line.encode('utf-8') for line in contents]
```

And then keep using writelines. This is how it always has been